src/main/java/Duke/Tasklist.java

```diff
@@ -0,0 +1,67 @@
+package Duke;
+
+import java.util.*;
+
+public class Tasklist {
+    private ArrayList<Task> tasks = new ArrayList<>();
+
+    /**
+     * Constructor for empty tasklist
+     */
+    Tasklist() {
+
+    }
+
+    /**
+     * Adds inputted tasks to the existing tasklist
+     * @param tasks to add
+     */
+    Tasklist(ArrayList<Task> tasks) {
+        this.tasks.addAll(tasks);
+    }
+
+    /**
+     * Add task to the list
+     * @param command on how to manipulate list
```

Comment:
The first word after @param is an exact copy of the parameter, a new sentence should start after this. For example '@param command Command on how to manipulate list'